### PR TITLE
Purchase Invoice Export

### PR DESCRIPTION
**Pull request by ruuushhh**

### CODE REVIEW

File: apps/business_central/exports/purchase_invoice/models.py [hunk 1]
```diff
@@ -13,26 +15,23 @@ class PurchaseInvoice(BaseExportModel):
...
-    accounting_export = models.OneToOneField(AccountingExport, on_delete=models.PROTECT, help_text='Accounting Export reference')
+    accounting_export = models.OneToOneField(AccountingExport, on_delete=models.PROTECT, help_text='Accounting Export reference', related_name='purchase_invoice')
     accounting_date = CustomDateTimeField(help_text='accounting date of purchase invoice')
     amount = FloatNullField(help_text='Total Amount of the invoice')
     code = StringNullField(max_length=10, help_text='unique code for invoice')
-    description = TextNotNullField(help_text='description for the invoice')
     invoice_date = CustomDateTimeField(help_text='date of invoice')
-    tax_amount = FloatNullField(help_text='total tax amount of the invoice')
-    vendor_id = StringNullField(help_text='id of vendor')
+    vendor_number = StringNullField(help_text='id of vendor')
```
Comment (NileshPant1999) [inline apps/business_central/exports/purchase_invoice/models.py:23]: why is this changed ??

File: apps/business_central/exports/purchase_invoice/queues.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                'status': 'ENQUEUED',
+                'type': 'PURCHASE_INVOICE'
+            }
+        )
+
+        if accounting_export.status not in ['IN_PROGRESS', 'ENQUEUED']:
+            accounting_export.status = 'ENQUEUED'
+            accounting_export.save()
+
+        """
```
Comment (NileshPant1999) [inline apps/business_central/exports/purchase_invoice/queues.py:38]: i have already added this todo in sage300 please to add here too

File: apps/business_central/exports/purchase_invoice/models.py [hunk 3]
```diff
@@ -13,26 +15,23 @@ class PurchaseInvoice(BaseExportModel):
...
     """
 
-    accounting_export = models.OneToOneField(AccountingExport, on_delete=models.PROTECT, help_text='Accounting Export reference')
+    accounting_export = models.OneToOneField(AccountingExport, on_delete=models.PROTECT, help_text='Accounting Export reference', related_name='purchase_invoice')
     accounting_date = CustomDateTimeField(help_text='accounting date of purchase invoice')
     amount = FloatNullField(help_text='Total Amount of the invoice')
     code = StringNullField(max_length=10, help_text='unique code for invoice')
-    description = TextNotNullField(help_text='description for the invoice')
     invoice_date = CustomDateTimeField(help_text='date of invoice')
-    tax_amount = FloatNullField(help_text='total tax amount of the invoice')
```
Comment (NileshPant1999) [inline apps/business_central/exports/purchase_invoice/models.py:22]: don't remove tax amount, we might support it later

**Review by NileshPant1999 — APPROVED** [review]

File: apps/business_central/exports/purchase_invoice/models.py (hunk 1 above)
Comment (ruuushhh) [reply apps/business_central/exports/purchase_invoice/models.py:23]: we don't send id we send number which is unique to dynamics
So to make it consistent i made the change

File: apps/business_central/exports/purchase_invoice/models.py (hunk 3 above)
Comment (ruuushhh) [reply apps/business_central/exports/purchase_invoice/models.py:22]: Will add

File: apps/business_central/exports/purchase_invoice/queues.py (hunk 2 above)
Comment (ruuushhh) [reply apps/business_central/exports/purchase_invoice/queues.py:38]: will add

File: apps/business_central/exports/purchase_invoice/models.py (hunk 3 above)
Comment (ruuushhh) [reply apps/business_central/exports/purchase_invoice/models.py:22]: will add this later as these are incremental PRs 
Will add when I flatten the migrations
